pkg/kds/context/context.go

```diff
@@ -68,8 +68,21 @@ func DefaultContext(
 		config_manager.ClusterIdConfigKey: true,
 	}
 
+	var globalSkipLabelOpts []util.CloneResourceMetaOpt
+	for _, prefix := range cfg.Multizone.Global.KDS.Labels.SkipPrefixes {
+		globalSkipLabelOpts = append(globalSkipLabelOpts, util.WithoutLabelPrefix(prefix))
+	}
+	var zoneSkipLabelOpts []util.CloneResourceMetaOpt
+	for _, prefix := range cfg.Multizone.Zone.KDS.Labels.SkipPrefixes {
+		zoneSkipLabelOpts = append(zoneSkipLabelOpts, util.WithoutLabelPrefix(prefix))
+	}
```

Comment:
nit: you could've probably simplify this code if `WithoutLabelPrefix` would accepts `prefixes []string`:
```go
UpdateResourceMeta(
    util.WithLabel(mesh_proto.ResourceOriginLabel, string(mesh_proto.GlobalResourceOrigin)),
    util.WithoutLabelPrefixes(cfg.Multizone.Zone.KDS.Labels.SkipPrefixes),
),
```

Reply:
indeed